lighthouse-viewer/app/src/main.js

```diff
@@ -43,3 +43,10 @@ Promise.all(loadPolyfillPromises).then(_ => {
   // eslint-disable-next-line no-new
   new LighthouseViewerReport();
 });
+
+if ('serviceWorker' in navigator) {
+  navigator.serviceWorker.register('/sw.js').catch(e => {
+    // eslint-disable-next-line no-console
+    console.error('Error during service worker registration:', e);
```

Comment:
actually, we probably want to register this at `${location.pathname}sw.js` so it is restricted to `googlechrome.github.io/lighthouse/viewer/` and also works on localhost.